Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/Settings/GetIshSetting.Tests.ps1

```diff
@@ -83,7 +83,9 @@ Describe “Get-IshSetting" -Tags "Read" {
 			$ishSession.DefaultRequestedMetadata = $oldDefaultRequestedMetadata
 			$ishFields.GetType().Name | Should BeExactly "Object[]"
 			if ((Get-IshVersion).MajorVersion -le 13) { $ishFields.Length | Should Be 19 }
-			if ((Get-IshVersion).MajorVersion -ge 14) { $ishFields.Length | Should Be 22 } # 14.0.0, fields FISHCOLLECTIVESPACESCFG, FISHPREVIEWRESOLUTION(value), FISHPREVIEWRESOLUTION(element) were added
+			if ((Get-IshVersion).MajorVersion -ge 14) { $ishFields.Length | Should Be 23 } 
+				# 14.0.0, fields FISHCOLLECTIVESPACESCFG, FISHPREVIEWRESOLUTION(value), FISHPREVIEWRESOLUTION(element) were added
```

Comment:
`(Get-IshVersion).MajorVersion` has other properties, you can tune using `(Get-IshVersion).RevisionVersion`

Reply:
Updated with RevisionVersion